scripts/windows/build/Publish-Branch.ps1

```diff
@@ -243,4 +252,46 @@ $LeafDeviceProjectFolder = Join-Path $BuildRepositoryLocalPath "smoke/LeafDevice
 	Write-Host
 if ($LASTEXITCODE -ne 0) {
 	throw "Failed publishing LeafDevice."
+}
+
+<#
+ # Publish IoTEdgeQuickstart
+ #>
+$IoTEdgeQuickstartProjectFolder = Join-Path $BuildRepositoryLocalPath "smoke/IotEdgeQuickstart"
+
+Write-Host "Publishing - IoTEdgeQuickstart x64"
```

Comment:
Nit: there's a lot of duplicate code in this section. A helper function would be nice...

Reply:
Good catch.

Reply:
added a variable for reuse